app/models/session.rb

```diff
@@ -1,5 +1,6 @@
 class Session < ApplicationRecord
-	validates_presence_of :title, :startDate
+	validates_presence_of :title, :start_date
```

Comment:
Same here, why did you change it back to snake case?

Reply:
Same above